### ISSUE TITLE
Chrome Buggy Behavior (potential defect) - Tab Groups being lost

### ISSUE THREAD

**Issue by Lcstyle** [Morikko/sync-tab-groups#80]
I just installed on Chrome Win10 X64 : Version 64.0.3282.167 (Official Build) (64-bit)
I am experiencing tab groups being lost.

Example Flow
1. Existing Window with buncha tabs
2. move buncha tabs to new window (new window contains all tabs for proposed new tab group)
3. enter name of new group in window where tabs have been moved to
4. Close Window
5. Return to "main" or remaining chrome browser window
6. newly created Tab Group (before closing it's window) is now Missing, and tabs are lost.

**Comment by Lcstyle** [comment]
Chrome just updated to Version 64.0.3282.186 (as opposed to .167), I haven't tried reproducing with this slightly updated version yet.  Will let you know, perhaps others can try and confirm.

**Comment by Morikko** [comment]
Good catch!

**Comment by Morikko** [comment]
Let me know if it is well corrected in next release.

**Comment by Lcstyle** [comment]
yep confirmed the fix in chrome. Thank you!

**Comment by Lcstyle** [comment]
Ack!  The original issue is fixed, but now, the parent window gets stuck with the name of the child window's group!  It's like both windows now are joined at the hip by the new tab group.  In this new use flow, I opened that second chrome window, gave it a name.  Then I simply navigated back to my main window, and clicked on the sync tab groups icon, and it showed up as the child window! (make visible was clicked) when my intention was to give them main window a different name.  If I then click / unclick group visible, the child group that I created previously gets cleared, and now both the parent and the child end up with the same default group name.

**Comment by Morikko** [comment]
I don't understand what really happens. Can you explain it differently ?
What do you mean with child window ?

**Comment by Lcstyle** [comment]
sorry I've been busy for a while.  Yes I can explain it differently, also I have input on #91 as well, I will look through all the tickets and provide repro instructions for ones I have seen in both Chrome and FF sometime this week.

**Comment by Lcstyle** [comment]
> What do you mean with child window ?

What I meant was, if you open Google Chrome you have an initial Single Window, let's call this MAIN, in the behavior I described above:
bq.  the parent window gets stuck with the name of the child window's group! It's like both windows now are joined at the hip by the new tab group.

Basically, I open Chrome and get my "main" window, then I open a new window, either by dragging a tab or just opening a new window manually.  I then go and name this new window in tab groups with some name.  When I navigate back to the MAIN window, it appears as though the name I gave the child or second window in the tab groups dialog is now the name of the Main or original window.
